src/adapter/z-stack/constants/common.ts

```diff
@@ -315,4 +196,89 @@ const COMMON: {
     }
 };
 
-export default COMMON;
\ No newline at end of file
+export enum ZnpCommandStatus {
+    SUCCESS = 0x00,
+    FAILURE = 0x01,
+    INVALID_PARAM = 0x02,
+    NV_ITEM_UNINIT = 0x09,
```

Comment:
Wouldn't `NV_ITEM_INITIALIZED` be a better name?

From the docs:
> 0x09 = Success, item created and initialized

Reply:
Good point, indeed. I copy-pasted the name from `SWRA442` doc. Apparently it is a typo in there. Renamed.